python/lsst/daf/butler/configs/datastores/formatters.yaml

```diff
@@ -40,7 +40,7 @@ ImageU: lsst.obs.base.formatters.fitsExposure.FitsImageFormatter
 DecoratedImageU: lsst.obs.base.formatters.fitsExposure.FitsExposureFormatter
 Mask: lsst.obs.base.formatters.fitsExposure.FitsMaskFormatter
 MaskX: lsst.obs.base.formatters.fitsExposure.FitsMaskFormatter
-MaskedImageF: lsst.obs.base.formatters.fitsMaskedImage.FitsMaskedImageFormatter
+MaskedImageF: lsst.obs.base.formatters.fitsExposure.FitsMaskedImageFormatter
```

Comment:
If you're moving this, the original needs to continue to work (basically forever) since this string is written into the butler database for datasets we've already written.  That makes me wonder if it's actually worthwhile to move it.

Reply:
I'm not moving it. I don't think this was every correct, since the formatter had been defined in fitsExposure.py.

https://github.com/lsst/obs_base/blob/main/python/lsst/obs/base/formatters/fitsExposure.py#L26

Reply:
Yup. I should perhaps update the PR title/ticket title?